packages/Frontend/src/components/App.jsx

```diff
@@ -10,14 +10,13 @@ import Library from './Library';
 import AuthenticateContainer from '../containers/AuthenticateContainer';
```

Comment:
We used to have a logout function that came from AppContainer because it was connected to redux, it was passed through this App component, then to the SidebarLeftOverlay so that a user could click logout and it would remove the user from redux. That's not necessary anymore with how firebase handled logouts, I'll make a comment in the sidebar left overlay with how logging out is handled now.